BuildResidentialHPXML/measure.rb

```diff
@@ -1659,6 +1659,36 @@ def arguments(model)
       args << arg
     end
 
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('lighting_fraction_cfl_interior', false)
+    arg.setDisplayName('Lighting: Fraction CFL Interior')
+    arg.setDescription('Fraction of all lamps (interior) that are compact fluorescent. Lighting not specified as CFL or LED is assumed to be incandescent.')
```

Comment:
Replace all "cfl" with "fluorescent" because it's _all_ fluorescent lighting (i.e., linear and compact). Remove any references to "compact".